releasenotes/notes/0.25/cx_cz_synthesis-3d5ec98372ce1608.yaml

```
@@ -1,10 +1,12 @@
 ---
-features:
+features_synthesis:
   - |
-    Added a new synthesis algorithm :func:`qiskit.synthesis.linear_phase.synth_cx_cz_depth_line_my`
-    of a CX circuit followed by a CZ circuit for linear nearest neighbor (LNN) connectivity in
-    2-qubit depth of at most 5n using CX and phase gates (S, Sdg or Z). The synthesis algorithm is
-    based on the paper of Maslov and Yang (https://arxiv.org/abs/2210.16195).
+    Added a new synthesis function :func:`~qiskit.synthesis.synth_cx_cz_depth_line_my`
+    which produces the circuit form of a CX circuit followed by a CZ circuit for linear
+    nearest neighbor (LNN) connectivity in 2-qubit depth of at most 5n, using CX and
+    phase gates (S, Sdg or Z). The synthesis algorithm is based on the paper of Maslov
+    and Yang (https://arxiv.org/abs/2210.16195).
```

Would it be nice to have these as links 🙂 
```suggestion
    phase gates (S, Sdg or Z). The synthesis algorithm is based on 
    `arXiv:2210.16195 <https://arxiv.org/abs/2210.16195>`__.
```